tests/jenkins/jobs/CreateGithubIssueExisting_Jenkinsfile.txt

```diff
@@ -5,4 +5,4 @@
             CreateGithubIssueExisting_Jenkinsfile.script(groovy.lang.Closure)
                CreateGithubIssueExisting_Jenkinsfile.createGithubIssue({message=[Error building OpenSearch, retry with: ./build.sh manifests/2.2.0/opensearch-2.2.0.yml --component OpenSearch --snapshot]})
                   createGithubIssue.readYaml({file=manifests/2.0.0/opensearch-2.0.0.yml})
-               CreateGithubIssueExisting_Jenkinsfile.echo(Exception occurred: java.io.FileNotFoundException: manifests/2.0.0/opensearch-2.0.0.yml (No such file or directory))
+               CreateGithubIssueExisting_Jenkinsfile.echo(Exception occurred: java.io.FileNotFoundException: manifests\2.0.0\opensearch-2.0.0.yml (The system cannot find the path specified))
```

Comment:
Looks like this is windows/linux path problem. Can you keep it default to linux for now?